server/data/words/german.json

```diff
@@ -435,6 +439,10 @@
     "word": "alt",
     "translation": "ancient"
   },
+  {
+	"word": "antik",
+	"translation": "ancient"  
```

Comment:
The translation for ancient seems to exist already. Could you please check if the translation is correct if not kindly update the same with the correct one.

Reply:
ancient can be translatet to both words so "alt" would also mean "old" and "antik" also could be translatet to old but is in most cases best translatet to ancient. Because old alredy is in the file i would suggest to just keep "antik".